_guides/cassandra.adoc

```diff
@@ -52,7 +52,7 @@ We recommend that you follow the instructions in the next sections and create th
 by step. However, you can go right to the completed example.
 
 The solution is located in the
-link:https://github.com/datastax/cassandra-quarkus/tree/master/quickstart[quickstart directory] of
```

Comment:
Actually this is already updated in https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/cassandra.adoc.

I am not sure when it is synch'ed?

Reply:
Ah yes, I see that. 

Is this a sign that the sync is not correctly happening? I would have expected it to happen after every release of Quarkus. So looking at https://github.com/quarkusio/quarkus/tags, I'd have expected the sync to happen with the 2.2.3.Final release 10 days ago.

Reply:
Depends if it has been backported or not.

Reply:
Ah yes, of course.

Should we just leave this change in, and let it be over-written when 2.3.0.Final is released? Or would you prefer I removed this and look for any similar cases?

Reply:
I'd prefer to have this change in a PR in the quarkusio repo with a backport label, but that's me :)

Reply:
But if 2.3.0 is coming soon maybe it's not necessary

Reply:
2.3.0.Final is estimated for Sept 29th, so it's soon enough IMO for this type of change. See: https://github.com/quarkusio/quarkus/wiki/Release-Planning